param/__init__.py

```diff
@@ -1287,6 +1287,112 @@ def __init__(self, **params):
         super(Array,self).__init__(ndarray, allow_None=True, **params)
 
 
+class DataFrame(ClassSelector):
+    """
+    Parameter whose value is a pandas DataFrame.
```

Comment:
Docstring explaining the arguments accepted?

Reply:
Addressed in 3dca9a1